src/environments/environment.ts

```diff
@@ -1,13 +1,15 @@
-import { environment as devEnvironment } from './environment.dev';
-import { environment as prodEnvironment } from './environment.prod';
-
-export interface Environment {
+interface Environment {
```

Comment:
Since this is the only file i this dir now, maybe we can move it to the higher level and remove the dir.

Reply:
But it's configured in this way in the project and moving it out causes problems